lib/src/components/NxTextInput/__tests__/NxTextInput.test.tsx

```diff
@@ -16,28 +16,22 @@ describe('NxTextInput', function() {
       },
       getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxTextInput, minimalProps);
 
-  it('renders a text input by default', function() {
-    expect(getShallowComponent()).toMatchSelector('input');
-    expect(getShallowComponent()).toHaveProp('type', 'text');
-    expect(getShallowComponent().find(NxTooltip)).not.toExist();
-  });
-
-  it('renders a text input wrapped in a NxTooltip if there are validation errors', function() {
-    expect(getShallowComponent({ validationErrors: 'foo' })).toMatchSelector(NxTooltip);
+  it('renders a text input wrapped in a NxTooltip by default', function() {
```

Comment:
It always does, right?

```suggestion
  it('renders a text input wrapped in a NxTooltip', function() {
```

Reply:
> It always does, right?

Oh I see, "text input" as opposed to "password input".